app/views/flood_risk_engine/shared/_address.html.erb

```diff
@@ -1,14 +1,14 @@
-<%= t('.postcode_hint') %>
+<%= form.t('.postcode_hint') %>
 <p>
```

Comment:
not entirely sure TBH, this is from the @reggieb version, and as it all worked as is  - without any locale changes to my stuff  - I assume it degrades  ok to generic stuff when no step yml defined